ui/helpers/constants/routes.ts

```diff
@@ -140,6 +140,10 @@ const PATH_NAME_MAP = {
   ///: END:ONLY_INCLUDE_IN
   [CONFIRM_ADD_SUGGESTED_NFT_ROUTE]: 'Confirm Add Suggested NFT Page',
   [CONNECT_HARDWARE_ROUTE]: 'Connect Hardware Wallet Page',
+  ///: BEGIN:ONLY_INCLUDE_IN(snaps)
+  [SNAPS_ROUTE]: 'Snap List Page',
```

Comment:
Maybe this, since it lists all the Snaps?
```suggestion
  [SNAPS_ROUTE]: 'Snaps List Page',
```